tola/forms.py

```diff
@@ -86,6 +86,9 @@ def clean_org(self):
 
     def __init__(self, *args, **kwargs):
         super(NewTolaUserRegistrationForm, self).__init__(*args, **kwargs)
+        if settings.DEFAULT_ORG and os.getenv('APP_BRANCH', '') == 'demo':
```

Comment:
Better `settings.APP_BRANCH == 'demo'`

Reply:
`APP_BRANCH` isn't a setting variable, it's an environment variable, so `settings.APP_BRANCH == 'demo'` doesn't work.